src/StreamingDataFrames/streamingdataframes/models/rows.py

```diff
@@ -6,7 +6,7 @@
 
 
 # TODO: add other dict functions like .get() , __contains__  and .copy()
-@dataclasses.dataclass(eq=True, kw_only=True, slots=True)
+@dataclasses.dataclass(eq=True)
```

Comment:
Since the `dataclasses` lib doesn't allow `slots=True` argument in 3.8, we need to manually specify the `__slots__`

Reply:
Removed all data classes and replaced them with a very basic class that enables simple equality checks on the slotted items (useful for a bunch of the unit tests we had)